README.md

```diff
@@ -1 +1,97 @@
-# Hell-Microverse-Project
\ No newline at end of file
+# HTML & CSS3 Course

```

Comment:
- [ ] kindly use this readme template. [readme](https://github.com/microverseinc/readme-template)

Reply:
> * [x]  kindly use this readme template. [readme](https://github.com/microverseinc/readme-template)
Ok i used a readme template

Reply:
@tufoinnkuo10 what do you mean by every comment prefix pls

Reply:
it is just a reminder that any comment/requested change with optional in it wont stop the approval of your pull request